L1Trigger/TrackFindingTracklet/interface/TrackletConfigBuilder.h

```diff
@@ -75,6 +78,17 @@ namespace trklet {
     //
     void buildProjections();
 
+#ifdef CMSSW_GIT_HASH
+    // Calculate phi range of modules read by each DTC.
+    void setDTCphirange(const tt::Setup* setup);
+
+    // Write DTC phi ranges to dtcphirange.txt for stand-alone emulation.
+    void writeDTCphirange() const;
+#else
+    // Set phi ranges after reading them from dtcphirange.txt
+    void setDTCphirange(const tt::Setup* setup = nullptr);
+#endif
```

Comment:
Does this code have to be inside the CMSSW_GIT_HASH? I would think you could just have:

void setDTCphirange(const tt::Setup* setup = nullptr);

and 

void writeDTCphirange() const;

Reply:
Your statement is true. Though we'd then have to replace https://github.com/cms-L1TK/cmssw/blob/L1TK-dev-12_0_0_pre4/L1Trigger/TrackFindingTracklet/src/TrackletConfigBuilder.cc#L1301 with:
#ifdef CMSSW_GIT_HASH
setDTCphirange(setup);
#else
setDTCphirange();
#endif 
so we wouldn't reduce the number of pragma statement. 

The existing pragma use you refer to has the advantages that: (i) It makes clear which functions are only needed to support stand-alone operation (making it easier to remove them when we eventually drop this); (ii) No CPU is used to compile the stand-alone functions when compiling for CMSSW.